src/entry.h

```diff
@@ -199,6 +199,7 @@ class Entry
     bool directoryGraph;      //!< do we need to draw the directory graph?
     bool collaborationGraph;  //!< do we need to draw the collaboration graph?
     CLASS_GRAPH_t inheritanceGraph; //!< type of inheritance graph?
+    bool inheritanceGraphExplicit;  //!< type of inheritance graph set implicit or explicit?
```

Comment:
I'd prefer the slightly longer`inheritanceGraphExplicitlySet` as a more explicit name.